### ISSUE TITLE
Any Option for adding reverse proxy

### ISSUE THREAD

**Issue by sujaykakkad** [luraproject/lura#415]
So I have a backend service running on a specific port and I want to proxy all the requests through krakend. My backend has around 10 different apis. So I want to know if there is any way to define reverse proxy or do I have to define all the 10 api routes in my krakend config file?

Also can krakend be used as a replacement for NGINX.

**Comment by kpacha** [comment]
krakend is an API Gateway, not a reverse proxy. you'll find all the reasoning here: https://www.krakend.io/blog/what-is-an-api-gateway/

**Comment by sujaykakkad** [comment]
So if I have a microservice which has 10 apis defined I will have to define all 10 in the json config file?

**Comment by kpacha** [comment]
I'd reword that statement to:

>if I have a client with 10 use cases, its gateway should define all 10 endpoints at the config as a contract to honor, regardless the number of endpoints the consumed backends expose

otherwise, your client will be heavily coupled to your backends instead of being coupled to a contract wrote and maintained just for it.

**Comment by sujaykakkad** [comment]
So what are differences and similarities between krakend and nginx?
